src/cljs/main/broadfcui/page/library/library_page.cljs

```diff
@@ -154,6 +154,14 @@
                        ((:update-aggregates props) aggregations)))
                    (on-done {:error status-text})))}))))))})
 
+(defn encode [text]
+  (clojure.string/escape text {\" "&quot;" \& "&amp;" \< "&lt;", \> "&gt;", \\ "&#x27;" \/ "&#x2F;"}))
```

Comment:
Maybe a short comment above this line about why these specific characters. I don't think quotes need escaping, but if they do, I'd imagine single quotes need to be there as well.